tests/storage/datavolume.go

```diff
@@ -1141,72 +1141,57 @@ var _ = Describe(SIG("DataVolume Integration", func() {
 		)
 	})
 
-	Context("With VirtualMachinePreference and PreferredStorageClassName", func() {
-		var vm *v1.VirtualMachine
-		var storageClass *storagev1.StorageClass
-		var virtualMachinePreference *instanceType.VirtualMachinePreference
-
-		BeforeEach(func() {
-			sc, exists := libstorage.GetRWOFileSystemStorageClass()
-			if !exists {
-				Fail("Fail test when Filesystem storage class is not present")
-			}
+	It("should always use VM defined storage class over PreferredStorageClassName", func() {
```

Comment:
Can't this convert to unit test too? It's fine if not in this PR, just curious, I don't see reason why not

Reply:
Yes we can.
Lets do it in a future PR. for now lets remove the covered test and then move the e2e into unit.